package-system/AWSNativeSDK/build_config.json

```diff
@@ -0,0 +1,25 @@
+{
+   "git_url":"https://github.com/aws/aws-sdk-cpp.git",
+   "git_tag":"1.7.167",
+   "package_name":"AWSNativeSDK",
+   "package_version":"1.7.167-rev3",
+   "package_url":"https://github.com/aws/aws-sdk-cpp",
+   "package_license":"Apache-2.0",
+   "package_license_file":"LICENSE",
+   "cmake_find_target":"FindAWSNativeSDK.cmake",
+   "cmake_find_template_custom_indent": 3,
+   "build_configs": ["Debug", "Release"],
+   "Platforms":{
+      "Windows":{
+         "Windows":{
+            "cmake_find_template":"FindAWSNativeSDK.cmake.Windows.template",
```

Comment:
not an issue with this PR because it was just updated yesterday, but the pull_and_build_from_git script now supports 'cmake_find_source' instead of 'cmake_find_template', which will not use the file as a template if you dont need any templatized parameters. That will make it so you dont have to escape the '$' with '$$' in the file.

Reply:
sounds good, will update in next PR